### PR TITLE
[Mri violations]Update react table mri violations

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/mri_violations/jsx/columnFormatter.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        resolution_status ='Inserted with flag';
+        break;
+        }
+
+      return <td className= {resolution_status_Style} style={fontColor}>
+                {resolution_status}
+             </td>;
+  }
+   if (column === "Problem" &&  row["Problem"] === "Protocol Violation" ) {
+  var patientname = row["PatientName"];
```
Comment (driusan) [inline modules/mri_violations/jsx/columnFormatter.js]: Can you run this file through eslint? The formatting looks off.

Comment (kongtiaowang) [inline modules/mri_violations/jsx/columnFormatter.js]: @driusan 
I have done with eslint format.

**Comment by driusan** [comment]
@kongtiaowang this has conflicts in the Gruntfile and needs to be rebased

**Comment by kongtiaowang** [comment]
@driusan 
the conflict file was fixed.

**Comment by kongtiaowang** [comment]
@alisterdev 
Could you review this?

**Comment by alisterdev** [comment]
@kongtiaowang Can you pull latest changes from 17.1, rebase this PR and fix stuff from review. 
We could retest and merge it then.

**Pull request closed without merging**